src/structures/APIMessage.js

```diff
@@ -184,6 +186,7 @@ class APIMessage {
       embeds,
       username,
       avatar_url: avatarURL,
+      allowed_mentions,
```

Comment:
All changes in this file can be simplified to just this now:
```suggestion
      allowed_mentions: this.options.allowedMentions,
```

Reply:
I'm trying to remember why I didn't already do that. I think this was with a view to adding a ClientOption which would set defaults? Thoughts on that?

Reply:
Defaults for what exactly?

Reply:
Similar to how ClientOptions#disableEveryone allows you to set "global" behaviour for all messages

Reply:
Wouldn't this be a matter of something like that then?
`allowed_mentions: this.options.allowedMentions || this.someHowGetClient().options.allowedMentions,`